tools/crm_resource_print.c

```diff
@@ -362,6 +362,7 @@ override_xml(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
+// Does not modify overrides or its contents
 PCMK__OUTPUT_ARGS("resource-agent-action", "int", "const char *", "const char *",
                   "const char *", "const char *", "const char *", "GHashTable *",
                   "crm_exit_t", "int", "const char *", "const char *", "const char *")
```

Comment:
Could we declare `overrides` static here and in the other places in this patch?

Reply:
I'm not sure exactly what you mean or what that would accomplish. We could make it global (rather than static -- i.e., file-scope). But we can't make it const, because the GLib functions require non-const.

Eventually we should have better doxygen, regardless.

Reply:
I meant const.  But yeah, doesn't matter because of glib.